boards/samr21-xpro/dist/samr21.jdebug

```diff
@@ -0,0 +1,254 @@
+
```

Comment:
Yeah I pointed that out as well, @poppe34 said it's from his debugger & he will be removing it.